nengo/utils/builder.py

```diff
@@ -27,7 +27,7 @@ def full_transform(conn, slice_pre=True, slice_post=True, allow_scalars=True):
         not using slicing, since these work fine in the reference builder.
         If false, these scalars will be turned into scaled identity matrices.
     """
-    transform = conn.transform
+    transform = conn.transform.init
```

Comment:
My guess is that this only works for Dense transforms? Maybe we should check/specify that?

Reply:
Done. As a sidenote, I did look at moving this into the `Dense` class like we talked about, but I saw that this function is only used in other `utils` functions.  So it seemed weird to move a relatively large chunk of code from utils into the core in order to support utils.  We could do it though if we find more common uses for `full_transform` .